ggml-cuda.cu

```diff
@@ -1347,10 +1347,30 @@ static void ggml_cuda_set_peer_access(const int n_tokens, int main_device) {
     GGML_UNUSED(main_device);
 }
 
+static cudaError_t cudaMemcpy2DPeerAsync(
```

Comment:
It's not good to "impersonate" the CUDA API, it can be confusing and it may cause conflicts in future CUDA versions. This should follow the same naming convention as the rest of the CUDA backend functions, eg. `ggml_cuda_xxx`.